src/FixedDataTableCell.js

```diff
@@ -338,6 +340,10 @@ class FixedDataTableCell extends React.Component {
     }
   };
 
+  _onColumnResizerDoubleClick = () => {
+    this.props.onColumnResizerDoubleClick(this.props.columnKey);
```

Comment:
Since this is essentially a wrapper over the `onDoubleClick` event handler, it's a good idea to also pass the original html `event` to the user.

Reply:
Sounds good, will add it